docs/git-resolver.md

```diff
@@ -55,11 +57,22 @@ for the name, namespace and defaults that the resolver ships with.
 
 ## Usage
 
-The `git` resolver has two modes: cloning a repository anonymously, or fetching individual files via an SCM provider's API using an API token.
+The `git` resolver has two modes: cloning a repository with `git clone` (with
```

Comment:
naming is hard, not sure if `git clone` method make sense, can use some suggestions